playbooks/roles

```diff
@@ -0,0 +1 @@
+../roles
```

Comment:
Why not moving roles as well ?

Reply:
Because, at least in theory, those roles could be useful by themself as well, and used by other projects, or tested in isolation.

I based this partially on the structure used in https://github.com/openshift/openshift-ansible

BTW, I'd reallly like to get rid of everything we do with `group_vars` (having a `group_vars` directory in the playbook, the symlink in `kubespray`,...)